.circleci/config.yml

```diff
@@ -760,43 +760,6 @@ workflows:
       - depupdate
       - build
       - test
-      - e2e-simple:
```

Comment:
Temporary change to reduce load on circleci while I debug the flaky MCP e2e tests.